global.json

```diff
@@ -1,6 +1,5 @@
 {
   "sdk": {
-    "rollForward": "latestFeature",
-    "version": "9.0.203"
```

Comment:
The SDK version is pinned for repeatability of CI, so please revert this change.

Reply:
That's my bad; sorry. I didn't mean to commit that.